src/e2e/java/uk/gov/hmcts/reform/sscs/functional/mya/BaseFunctionTest.java

```diff
@@ -0,0 +1,112 @@
+package uk.gov.hmcts.reform.sscs.functional.mya;
+
+import static java.lang.Long.valueOf;
+import static org.apache.http.conn.ssl.SSLConnectionSocketFactory.ALLOW_ALL_HOSTNAME_VERIFIER;
+
+import java.io.IOException;
+import java.security.KeyManagementException;
+import java.security.KeyStoreException;
+import java.security.NoSuchAlgorithmException;
+import java.util.function.Supplier;
+import lombok.extern.slf4j.Slf4j;
+import org.apache.http.HttpHost;
+import org.apache.http.client.HttpClient;
+import org.apache.http.conn.ssl.SSLConnectionSocketFactory;
+import org.apache.http.conn.ssl.TrustSelfSignedStrategy;
+import org.apache.http.impl.client.CloseableHttpClient;
+import org.apache.http.impl.client.HttpClientBuilder;
+import org.apache.http.ssl.SSLContextBuilder;
+import org.junit.Before;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.junit4.SpringRunner;
+import uk.gov.hmcts.reform.sscs.TribunalsCaseApiApplication;
+import uk.gov.hmcts.reform.sscs.ccd.domain.SscsCaseDetails;
+import uk.gov.hmcts.reform.sscs.ccd.service.CcdService;
+import uk.gov.hmcts.reform.sscs.idam.IdamService;
+
+@RunWith(SpringRunner.class)
+@SpringBootTest(classes = {TribunalsCaseApiApplication.class, CorIdamService.class})
+@Slf4j
+public abstract class BaseFunctionTest {
+    private final String baseUrl = System.getenv("TEST_URL") != null ? System.getenv("TEST_URL") : "http://localhost:8090";
```

Comment:
Would be good if we could have the Tribunals tests running on same port. Maybe move the other tribunals tests over to 8090 if it is easy to do (As 8080 seems to clash with of other services)